src/main/java/com/GeorgesServer/app/EstablishesConnection.java

```diff
@@ -1,19 +1,27 @@
 package com.GeorgesServer.app;
 
-import java.io.IOException;
+import java.io.*;
 import java.net.ServerSocket;
 import java.net.Socket;
+import java.nio.Buffer;
 
 public class EstablishesConnection {
 
+    private SocketFactory socketFactory;
+
+    public EstablishesConnection (SocketFactory socketFactory) {
```

Comment:
We have two different naming conventions. you're going to have to pick one. choose between something like EstablishesConnection and RequestReader.